addons/website_hr_recruitment/static/src/scss/website_hr_recruitment.scss

```diff
@@ -1,24 +1,89 @@
-.o_website_hr_recruitment_jobs_list {
-    #jobs_grid_left {
-        ul.flex-column > li.nav-item > a {
-            padding: .5rem 1rem;
-            border-radius: 0.125rem;
+@mixin o_jobs_has_shadows {
+    &.o_jobs_has_shadows {
+        &, * {
+            transition: all .3s;
         }
-    }
-    .card {
-        padding: 15px;
-        box-shadow: 0 1px 1px 0 rgba(0, 0, 0, 0.17);
-
         &:hover {
-            transition: box-shadow 0.3s cubic-bezier(0.55, 0, 0.1, 1);
             box-shadow: 0 6px 10px 0 rgba(0, 0, 0, 0.1), 0 2px 2px 0 rgba(0, 0, 0, 0.05);
         }
+    }
+}
+@mixin o_jobs_has_color {
+    &.o_jobs_has_color {
+        &, * {
+            transition: all .15s;
+        }
+        &:hover {
+            * {
+                color: theme-color('primary') !important;
+            }
+            border: solid 1px theme-color('primary');
+        }
+    }
+}
+
+.o_website_hr_recruitment_jobs_list {
+    #jobs_grid {
         .o_job_infos {
             .fa {
                 color: theme-color('primary');
                 margin-right: 8px;
             }
         }
+        button[aria-pressed="false"] {
+            &::after {
+                content: "Read more...";
+            }
+        }
+        button[aria-pressed="true"] {
+            &::after {
+                content: "Read less...";
+            }
+            ~ .o_truncate {
+                max-height: 9999px;
+                &::after {
+                    opacity: 0;
+                }
+            }
+        }
+        .o_truncate {
+            order: -1;
+            max-height: 100px;
+            overflow: hidden;
+            position: relative;
+            &, * {
+                transition: all 1s;
+            }
+            &::after {
+                position: absolute;
+                bottom: 0;
+                height: 100%;
+                width: 100%;
+                content: "";
+                background: linear-gradient(to top,rgba(255,255,255, 1) 20%, rgba(255,255,255, 0) 80%);
+                pointer-events: none;
+                opacity: 1;
+            }
+        }
+        .card {
+            padding: 15px;
```

Comment:
Better to use default bootstrap classes (eg. `card-body`).
If it's not possible at last use the default grid metric (16px)